liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -116,15 +114,20 @@ protected void installLooseConfigApp() throws Exception {
             // referencing dependent library jar file from mvn repository or set
             // loose application configuration reference to dependent eclipse project output classpath
             if (eclipseModules.isEmpty()) {
-                addLibraryFromM2(libraries, config);
+                addLibraries(libraries, config);
             } else {
                 for (Artifact library : libraries) {
-                    if (library.getFile() != null && eclipseModules.contains(library.getFile().getName())) {
-                        MavenProject module = getSiblingModule(library);
-                        config.addArchive(module.getBuild().getOutputDirectory(),
-                                "/WEB-INF/lib/" + module.getBuild().getFinalName() + "." + module.getPackaging());
-                    } else {
+                    if (library.getFile() == null || !eclipseModules.contains(getFileName(library.getFile()))) {
                         addLibraryFromM2(library, config);
+                    } else {
+                        File classDir = new File(project.getBasedir() + "/../" + library.getArtifactId() + "/target/classes");
+                        log.debug("sibling module target class directory pathname: " + classDir.getCanonicalPath());
+                        
+                        if (classDir.exists()) {
```

Comment:
For command line scenario, library.getFile() will always return jar file from Maven local (m2) repository.  The library.getArtifactId() give me the hint of the directory of the project.  If it is a sibling module project, it will be project.getBasedir() + "/../" + library.getArtifactId().  If I can find classDir, I can set looseConfig to its output directory.  If not, I will just use the jar file from m2 repository.